CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+# Contributing to Armbian
+
+We'd love to have you join the Armbian community! Below summarizes the processes that we follow.
+
+## Reporting Issues
+
+Before reporting an issue, check our backlog of open issues to see if someone else has already reported it. If so, feel free to add your scenario, or additional information, to the discussion. Or simply "subscribe" to it to be notified when it is updated.
+
+If you find a new issue with the project we'd love to hear about it! The most important aspect of a bug report is that it includes enough information for us to reproduce it. So, please include as much detail as possible and try to remove the extra stuff that doesn't really relate to the issue itself. The easier it is for us to reproduce it, the faster it'll be fixed!
+
+Please don't include any private/sensitive information in your issue!
+
+## Working On Issues
+
+Once you have decided to contribute to Armbian by working on an issue, check our backlog of open (or our [JIRA](https://armbian.atlassian.net)) looking for any that do not have an "In Progress" label attached to it. Often issues will be assigned to someone, to be worked on at a later time. If you have the time to work on the issue now add yourself as an assignee, and set the "In Progress" label if you’re a member of the “Containers” GitHub organization. If you can not set the label, just add a quick comment in the issue asking that the “In Progress” label be set and a member will do so for you.
+
+## Contributing
+
+This section describes how to start a contribution to Armbian.
+
+### Prepare your environment
+
+* Create an Ubuntu 20.04 VM with VirtualBox or other hypervisor.
```

Comment:
Officially supported build environment is 22.04 only. So not sure what this is about.

Reply:
sorry just copy paste from [here](https://github.com/sgjava/patchbian/blob/main/README.md), fix it